app/src/main/AndroidManifest.xml

```diff
@@ -7,18 +7,26 @@
     android:allowBackup="true"
     android:icon="@mipmap/ic_launcher"
     android:label="@string/app_name"
+    android:name="com.example.myapplication.OppiaApplication"
```

Comment:
Why do we need this?

Reply:
Its used for logging system, to initialize hyperlog I created application class which is initialized only once throughout the app.